### PR TITLE
ID-4347 [Feat] Updated docs for analytics improvements

### PR DESCRIPTION
**Pull request by Arpanexe**
### Product areas affected

CLI -> `docs\API\core\analytics.md`

### What does this PR do?

This PR updates analytics docs for the analytics improvements

### JIRA ticket

[ID-4347](https://weboo.atlassian.net/browse/ID-4347)

[ID-4347]: https://weboo.atlassian.net/browse/ID-4347?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/API/core/analytics.md [hunk 1]
```diff
@@ -256,3 +256,92 @@ Fliplet.Apps.Analytics.count(appId, { group: [ 'data._userEmail' ] }).then(funct
...
+    "source": "production",
+    "from": "2024-05-01",
+    "to": "2024-05-31",
+    "group":"app",
+    "sum": ['sentEmails', 'sentSMS', 'sentPushNotifications', 'totalDevices', 'totalSessionDuration'],
+}).then(function (data) {
+  // console.log(data)
+});
+```
+- `source`: can be either `production` or `master`. production will be used to fetch data for the live app
```
Comment (debankurghosh2061) [inline docs/API/core/analytics.md:281]: "Production" Should be capital

File: docs/API/core/analytics.md [hunk 2]
```diff
@@ -256,3 +256,92 @@ Fliplet.Apps.Analytics.count(appId, { group: [ 'data._userEmail' ] }).then(funct
...
+- sentEmails
+- sentSMS
+- sentPushNotifications
+- totalSessionDuration
+
+#### Group type `user`
+
+This will return aggregated sum for each user, below are the possible values for `sum`
+
+- `totalDevices
```
Comment (debankurghosh2061) [inline docs/API/core/analytics.md:307]: Foe all of this list why is there a quote in front of these

File: docs/API/core/analytics.md (hunk 1 above)
Comment (Arpanexe) [reply docs/API/core/analytics.md:281]: @debankurghosh2061 We use camel case for property and variable names in our API. Therefore, they need to be passed in this format.

File: docs/API/core/analytics.md (hunk 2 above)
Comment (Arpanexe) [reply docs/API/core/analytics.md]: @debankurghosh2061 Removed unnecessary quotes

File: docs/API/core/analytics.md (hunk 1 above)
Comment (debankurghosh2061) [reply docs/API/core/analytics.md:281]: In that case, I would put 'production' in the quotes as well like you have in the first sentence.

**Review by debankurghosh2061 — APPROVED** [review]
Agreed with the changes.

**Pull request merged**